README.md

```diff
@@ -41,6 +43,34 @@ sudo ln -s ~/.mason/mason /usr/local/bin/mason
 
 The second line is optional.
 
+#### Submodule
+
+Mason can be added a submodule to your repository instead of creating a global symlink. This is helpful for other contributors to get set up quickly. Make sure to include the final part of the following command `.mason/` so your submodule path has the leading `.` instead of just being `mason/`.
+
+```bash
+git submodule add git@github.com:mapbox/mason.git .mason/
+```
+        
+This will append a few lines to your `.gitmodules` file. Something like this:
+
+```
+[submodule ".mason"]
+    path = .mason
+    url = git@github.com:mapbox/mason.git
+```
+   
+Update your `Makefile` to point to the mason scripts and provide an installation script for the necessary dependencies. The following installs two Mason packages with the `make mason_packages` command.
+
+```Make
+MASON ?= .mason/mason
+
```

Comment:
Adding:

```
$(MASON):
	git submodule update --init
```

Will be good to trigger the submodule init if `.mason/mason` is not found.